cli/pkg/local/app.go

```diff
@@ -122,12 +125,15 @@ func NewApp(ctx context.Context, ver version.Version, verbose bool, olapDriver,
 		olapDSN = path.Join(projectPath, olapDSN)
 	}
 
+	if len(split) > 1 {
+		projectPath = projectPath + `|` + split[1]
+	}
 	// Create instance with its repo set to the project directory
 	inst := &drivers.Instance{
 		ID:           DefaultInstanceID,
 		OLAPDriver:   olapDriver,
 		OLAPDSN:      olapDSN,
-		RepoDriver:   "file",
+		RepoDriver:   "git",
```

Comment:
The changes in `cli/pkg/local/app.go` can be omitted since this driver will only be used in cloud. On local, people will do the `clone` separately and use the `file` connector.